web/crux-ui/src/models/node.ts

```diff
@@ -50,12 +50,17 @@ export type NodeDetails = DyoNode & {
   inUse: boolean
 }
 
-export const nodeConnectionOf = (node: DyoNode): NodeConnection => ({
-  address: node.address,
-  status: node.status,
-  connectedAt: node.connectedAt,
-  version: node.version,
-})
+export const nodeConnectionOf = (node: DyoNode): NodeConnection =>
```

Comment:
Node shouldn't be null, this code is only required in darklens.
You can revert this to the original code.